pkg/apis/apiserver/organization_REST.go

```diff
@@ -350,7 +360,45 @@ func internalErrorWatchEvent(err error) watch.Event {
 	}
 }
 
-func (o *OrganizationREST) isVisible(ctx context.Context, organization *Organization) (bool, error) {
+func (o *OrganizationREST) checkIsOwner(ctx context.Context, organization *Organization) error {
+	attrs, err := filters.GetAuthorizerAttributes(ctx)
+	if err != nil {
+		return err
+	}
+	visible, err := o.isMember(ctx, organization)
+	if err != nil {
+		return err
+	}
+	if !visible {
+		return apierrors.NewNotFound(qualifiedResource, organization.Name)
+	}
+	isOwner, err := o.containsUser(ctx, organization.Spec.Owners)
+	if err != nil {
+		return err
+	}
+	if !isOwner {
+		return apierrors.NewForbidden(
+			qualifiedResource,
+			organization.Name,
+			fmt.Errorf("organization ownership is required for %s operation", attrs.GetVerb()),
+		)
+	}
+	return nil
+}
+
+func (o *OrganizationREST) isMember(ctx context.Context, organization *Organization) (bool, error) {
+	return o.containsUser(ctx,
+		append(
+			// This is important for seeing organizations you own before controller syncs status
+			// otherwise a watch misses create event
+			organization.Spec.Owners,
+			organization.Status.Members...,
+		),
+	)
+}
+
+// containsUser function should only be called in the `isMember` and `isOwner`, not in the REST implementations, having `isMember` `containsUser` `checkIsOwner` all in REST operations just confuses people
```

Comment:
Have you really read this comment before you copied it? I am still seeing that `containsUser` is being used in `Create`

Reply:
and there's a comment in create why is that so. Have you read that comment?

Reply:
Technically I could abuse the `isMember` but this is more explicit in what it's doing

Reply:
> and there's a comment in create why is that so. Have you read that comment?

Yes, of course I have read it. If you pay more attention on my previous review message: https://github.com/kubermatic/bulward/pull/43#pullrequestreview-447239986, the first line in the code block:
`// This function can only error, but I noticed your comment in `Create` that you want to return different errors, so the bool can help you with that.`
proves that I have read you comments.

The reason that I don't think you have read the comment before writing the comment is that it mentions that `only be called in isMember and isOwner`, but it is still called in `Create`. If you don't agree with the idea, you can still call it in `Create`,but we should make proper comments about the usage of functions, and we should not introduce contradictions like this in our project.